src/Application.php

```diff
@@ -921,8 +921,6 @@ public function prepareForConsoleCommand($aliases = true)
      * Get the application namespace.
      *
      * @return string
-     *
-     * @throws \RuntimeException
```

Comment:
Please leave this fix to the master branch (if it was not already made there). The additional throw I think are fine for 8.x.